packages/block-library/src/navigation-menu/index.php

```diff
@@ -49,20 +49,14 @@ function build_css_colors( $attributes ) {
  * @return string Returns the post content with the legacy widget added.
  */
 function render_block_navigation_menu( $attributes, $content, $block ) {
-	$colors          = build_css_colors( $attributes );
-	$class_attribute = sprintf( ' class="%s"', esc_attr( $colors['css_classes'] ? 'wp-block-navigation-menu ' . $colors['css_classes'] : 'wp-block-navigation-menu' ) );
-	$style_attribute = $colors['inline_styles'] ? sprintf( ' style="%s"', esc_attr( $colors['inline_styles'] ) ) : '';
+  $colors  = build_css_colors( $attributes );
```

Comment:
Is this function `build_css_colors` not intended to be namespaced / prefixed specific to the plugin / block / WordPress? (i.e. is it meant to be for general usage?)

Reply:
Is there a namespacing standard for PHP functions in Gutenberg?

Reply:
> Is there a namespacing standard for PHP functions in Gutenberg?

Now that you mention it, not explicitly.

For most functions, we follow [best practices](https://developer.wordpress.org/plugins/plugin-basics/best-practices/#prefix-everything) in using a `gutenberg_` prefix for plugin-specific functions.

It's a little trickier for these blocks though, since they're [copied verbatim](https://github.com/WordPress/wordpress-develop/blob/e879f7d45723a01ac94fe2fa28d68c39194f8772/tools/webpack/packages.js#L96-L108) into core. So the concern is in considering that, should this be merged to core, these become new global WordPress functions.

Most other dynamic blocks follow some convention of `render_block_foo` and `register_block_foo`, but there's definitely some (unfortunate) ad hoc naming for similar utility functions ([example](https://github.com/WordPress/gutenberg/blob/fbd7fb80d0ad67a7ccee3227b772a7ee56f47ea6/packages/block-library/src/categories/index.php#L68)).

It's definitely not a blocker for merging this pull request, since it wasn't introduced here, but we should see about establishing a convention here that makes sense.

Reply:
`namspace Gutenberg;` 😱